### ISSUE TITLE
aspectRatio bug

### ISSUE THREAD

**Issue by J0ker98** [SDWebImage/SDWebImage#3016]
### New Issue Checklist

* [x] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [x] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [x] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name          | ios
 Platform Version       | 13.4
 SDWebImage Version     | latest
 Integration Method     | cocoapods
 Xcode Version           | Xcode 11
 Repro rate              | all the time (100%)
 Repro with our demo  | n/a
 Demo project link       | n/a

### Issue Description and Steps

I'm using SDWebImage in SwiftUI and when I have two images, one with `.aspectRatio(contentMode: .fit)` and another with `.aspectRatio(contentMode: .fill)`, the second image aspectRatio is not being set correctly.

Before: https://ibb.co/DKfD861
After: https://ibb.co/sb4mwvj

**Comment by J0ker98** [comment]
Ops, opened this in the wrong repo
